javascript/index.html

```diff
@@ -73,6 +73,20 @@ <h4>
                     to the local variable <code>self</code> remains in scope and is accessible there.
                 </p>
             </li>
+            <li>
```

Comment:
Please shift this into another file, do take a look at the fresh master branch code

Reply:
Done sir